npm ERR! install Couldn't read dependencies

When running on home (`/home/safarov/`) gives the following error for _Email on npm_. The rest works fine.

```
Email on npm
npm ERR! install Couldn't read dependencies
npm ERR! Error: ENOENT, open '/home/safarov/package.json'
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Linux 3.13.0-52-generic
npm ERR! command "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! cwd /home/safarov
npm ERR! node -v v0.10.25
npm ERR! npm -v 1.3.10
npm ERR! path /home/safarov/package.json
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /home/safarov/npm-debug.log
npm ERR! not ok code 0
/usr/local/bin/github-email: line 37: ./node_modules/.bin/npm-email: No such file or directory
```

OS: Ubuntu 14.04

--- a/github-email.sh
+++ b/github-email.sh
@@ -24,15 +24,18 @@ log() {
 user="$1"
 repo="$2"
 
+sourceFile=$(node -e "console.log(require('fs').realpathSync('$0'))")
+sourceDir=$(dirname $sourceFile)
+
 log 'Email on GitHub'
 curl "https://api.github.com/users/$user" -s \
     | sed -nE 's#^.*"email": "([^"]+)",.*$#\1#p'
 
 
 log 'Email on npm'
-npmEmail="./node_modules/.bin/npm-email"
+npmEmail="$sourceDir/node_modules/.bin/npm-email"
 if [ ! -f $npmEmail ]; then
-	npm install >/dev/null
+	(cd $sourceDir && npm install >/dev/null)
 fi
 eval $npmEmail $user
 

